merge.go

```diff
@@ -6,6 +6,13 @@ import (
 	"strings"
 )
 
+type MergeSetConfigType string
+
+const (
+	CREATE MergeSetConfigType = "CREATE"
+	MERGE  MergeSetConfigType = "MERGE"
+)
+
```

Comment:
The Neo4j docs doesn't use the term "merge" to refer to the `+=` operator, so I find that confusing.
```suggestion
```

Reply:
Removed.